app/views/steps/capital/investments/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= t('steps.capital.caption') %></span>
+
+    <%= govuk_error_summary(@form_object) %>
+
+    <h1 class="govuk-heading-xl">
+      <%= t(".heading.#{@form_object.investment_type}") %>
+    </h1>
+
```

Comment:
The design has a <p> here 
`You told us your client has investments` 🤷‍♂️ 

https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=8762-121907&mode=design&t=bhX1SNEEzsjD4Rcj-0

Reply:
Great spot! Thank you :)